package.json

```
@@ -1,8 +1,9 @@
 {
   "name": "svelte-state-renderer",
-  "version": "3.3.0",
+  "version": "4.0.0",
```

I'll definitely push this out as a breaking version bump, but my deploy script changes the version here automatically
```suggestion
  "version": "3.3.0",
```

---

Thanks!